src/microbe_stage/PlacedOrganelle.cs

```diff
@@ -147,6 +162,9 @@ public float StorageCapacity
         }
     }
 
+    [JsonIgnore]
```

Comment:
Should this be actually saved? I'm thinking about future use where we might want to have the enzymes build up or something like that. For now it doesn't matter but would make save upgrade probably easier in the future.

Reply:
Made it savable.